src/app/Http/Controllers/Operations/FetchOperation.php

```diff
@@ -60,13 +60,13 @@ private function fetch($arg)
         $config['searchable_attributes'] = $config['searchable_attributes'] ?? $model_instance->identifiableAttribute();
         $config['query'] = isset($config['query']) && is_callable($config['query']) ? $config['query']($config['model']) : $model_instance; // if a closure that has been passed as "query", use the closure - otherwise use the model
 
-        // FetchOperation is aware of an optional parameter 'keys' that will fetch you the entity/entities that match the provided keys
```

Comment:
Hmm why did you change ```key``` to ```related_keys``` here?

Reply:
Just to be clear on what those keys mean. They are model related keys, so I just updated the name to reflect that. `keys` seemed very vague.

Reply:
Oh ok. 

Let's keep `keys` - I think it's clear enough. At this point (inside FetchOperation, or inside the JS) we only work with that "related item", so of course they're that model's keys. They're the keys of the only model we're working with. At least that's what I think now - I reserve the option to say "_I was wrong, you were right_" in the future 😆